test/unit/webdriver/touch_action_test.py

```diff
@@ -37,6 +37,67 @@ def test_tap_x_y_json(self, touch_action):
         touch_action.tap(ElementStub(2), 3, 4)
         assert json == touch_action.json_wire_gestures
 
+    def test_press_json(self, touch_action):
+        json = [
+            {'action': 'press', 'options': {'element': 3}}
+        ]
+        touch_action.press(ElementStub(3))
+        assert json == touch_action.json_wire_gestures
+
+    def test_press_x_y_json(self, touch_action):
+        json = [
+            {'action': 'press', 'options': {'element': 4, 'x': 3, 'y': 4}}
+        ]
+        touch_action.press(ElementStub(4), 3, 4)
+        assert json == touch_action.json_wire_gestures
+
+    def test_long_press_json(self, touch_action):
+        json = [
+            {'action': 'longPress', 'options': {'element': 5, 'duration': 2000}}
+        ]
+        touch_action.long_press(ElementStub(5), duration=2000)
+        assert json == touch_action.json_wire_gestures
+
+    def test_long_press_x_y_json(self, touch_action):
+        json = [
+            {'action': 'longPress', 'options': {'element': 6, 'x': 3, 'y': 4, 'duration': 1000}}
+        ]
+        touch_action.long_press(ElementStub(6), 3, 4)
+        assert json == touch_action.json_wire_gestures
+
+    def test_wait_json(self, touch_action):
+        json = [
+            {'action': 'wait', 'options': {'ms': 10}}
+        ]
+        touch_action.wait(10)
+        assert json == touch_action.json_wire_gestures
+
+    def test_wait_without_ms_json(self, touch_action):
+        json = [
+            {'action': 'wait', 'options': {'ms': 0}}
+        ]
+        touch_action.wait()
+        assert json == touch_action.json_wire_gestures
+
+    def test_move_to_json(self, touch_action):
+        json = [
+            {'action': 'moveTo', 'options': {'element': 7, 'x': 3, 'y': 4}}
+        ]
+        touch_action.move_to(ElementStub(7), 3, 4)
+        assert json == touch_action.json_wire_gestures
+
+    def test_release_json(self, touch_action):
+        json = [
+            {'action': 'tap', 'options': {'count': 1, 'element': 8}}, {'action': 'release', 'options': {}}
+        ]
+        touch_action.tap(ElementStub(8)).release()
+        assert json == touch_action.json_wire_gestures
+
+    def test_perform_json(self, touch_action):
+        json = []
+        touch_action.tap(ElementStub(9)).perform()
+        assert json == touch_action.json_wire_gestures
```

Comment:
You can write as below if you would like to make sure the `json` is exactly cleared after the `perform`

```
json_tap = [...]
touch_action.tap(ElementStub(9))
assert json_tap == touch_action.json_wire_gestures
touch_action.perform()
assert [] == touch_action.json_wire_gestures
```